src/components/bottom_bar/bottom_bar.xml

```diff
@@ -13,7 +13,7 @@
           <t t-call="o-spreadsheet-Icon.PLUS"/>
         </div>
       </Ripple>
-      <Ripple>
```

Comment:
Not sure if it's the "correct" way to fix the problem. I've though of some alternatives, but I'm nort sure what's the best.

1. Stop the event. This works very well, but is easy to forget to do, and what if we also want to handle the event in the bottom bar?
2. Add custom payload to the click event, so `Menu` know that it should close itself. Also easy to forget, and need a bit of typing.
3. Add the "parent element" in the props of the `Menu`. 
4. Make sure to always unmount then recreate a `Menu`. Not owly at all.

I like 2)/3) the best I think, but 1) is also very simple and have no problem for the moment.

Thoughts ?

Reply:
5. Use `capture: true` in the event listener of Menu.